v63005/outref/gtm8877.txt

```diff
@@ -0,0 +1,30 @@
+# NO RESTRICTIONS
+original input
+original input
+
+# FILTER RETURNING -1
+%YDB-E-COMMFILTERERR, Error executing the command filter for gtmzsystem. RESTRICTEDOP
+%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: echo 'original input'
+%YDB-E-COMMFILTERERR, Error executing the command filter for gtmpipeopen. RESTRICTEDOP
+%YDB-E-RESTRICTEDOP, Attempt to perform a restricted operation: echo 'original input'
+ZSTATUS=pipeopenfn+4^gtm8877,%YDB-E-IONOTOPEN, Attempt to USE an I/O device which has not been opened
+
+# FILTER RETURNING A STRING
+filtered output
+filtered output
+
+# FILTER INVOCATION IN A TP TRANSACTION
```

Comment:
Can we have an additional test of FILTER INVOCATION IN A TP TRANSACTION WITH RESTARTS (the restarts can be induced by a zsystem $incr just like was done in v63002/gtm8165 for the tpnotacid case.